extensions/redis-client/deployment/src/main/java/io/quarkus/redis/client/deployment/DevServicesProcessor.java

```diff
@@ -44,9 +56,9 @@
     @Produce(ServiceStartBuildItem.class)
     @BuildStep(onlyIfNot = IsNormal.class, onlyIf = IsDockerRunningSilent.class)
     public void startRedisContainers(LaunchModeBuildItem launchMode,
-            BuildProducer<RunTimeConfigurationDefaultBuildItem> runTimeConfiguration,
+                                     BuildProducer<RunTimeConfigurationDefaultBuildItem> runTimeConfiguration,
```

Comment:
Why these spaces?

Reply:
I believe that this was autoformatted by my IntelliJ to align it with the other params.
So the `runTimeConfiguration` is aligned with `launchMode` above.

I can revert it as it's just formatting.

Reply:
`mvn clean compile` should fix that automatically (using the Quarkus formatter style).

Reply:
Fixed.